interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -16,6 +16,67 @@
 require_once("../../globals.php");
 require_once($GLOBALS['srcdir']."/options.inc.php");
 
+
+//Function do translate from another language to English
+if (!(function_exists('ltox'))) {
+
+    /**
+     * @param $wordInLang
+     * @param bool $oneWord
+     * @return bool or English text of var
+     */
+    function ltox( $wordInLang, $oneWord = true )
+    {
+        //$oneWord this variable for future using in case if we are need search by full string
+        if( strlen($wordInLang) <=0 ) return false;
+
+        // set language id
+        if (!empty($_SESSION['language_choice'])) {
+            $lang_id = $_SESSION['language_choice'];
+        } else {
+            $lang_id = 1;
+        }
+
+        if( $lang_id != 1 ){
+            $sql = "SELECT constant_name FROM lang_constants WHERE cons_id = (SELECT cons_id FROM lang_definitions WHERE lower(definition) REGEXP '^".mb_strtolower($wordInLang)."[^][[:space:]]*$'  LIMIT 1)";
+            $res = sqlStatement($sql);
+            if ($row = sqlFetchArray($res)) {
+                $wordInLang = $row["constant_name"];
+            }
+        }
+
+        return $wordInLang;
+    }
+}
+//Function partiality translate date to need format
+if(!(function_exists('DateToYYYYMMDD_Partial'))){
+    function DateToYYYYMMDD_Partial($DateValue)
```

Comment:
Do none of the functions here work for what you are trying to do:
https://github.com/openemr/openemr/blob/master/library/formatting.inc.php

Reply:
Actually, for the DOB column, wouldn't it be better to use the date widget(which supports internationalized dates)? And then you catch it with the DateToYYYYMMDD() function.

Reply:
And I suppose if you wish to support partial dates, then could make a DateToYYYYMMDD_Partial in the formatting.inc.php that is based on this function: https://github.com/openemr/openemr/blob/master/library/formatting.inc.php#L198
(and I would put it in that library)

Reply:
Its a nice idea. I'm didn't write this function in file "formatting.inc.php", without your agreement.